docs/backend/quantization.md

```diff
@@ -74,24 +70,64 @@ autoround.quantize_and_save(quant_path, format=format) # quantize and save
 # for VLMs
 from auto_round import AutoRoundMLLM
 from transformers import Qwen2VLForConditionalGeneration, AutoProcessor, AutoTokenizer
-
 model_name = "Qwen/Qwen2-VL-2B-Instruct" 
 quant_path = "Qwen2-VL-2B-Instruct-autoround-4bit"
 model = Qwen2VLForConditionalGeneration.from_pretrained(
     model_name, trust_remote_code=True, torch_dtype="auto")
 tokenizer = AutoTokenizer.from_pretrained(model_name)
 processor = AutoProcessor.from_pretrained(model_name, trust_remote_code=True)
-
-## quantize the model
 bits, group_size, sym = 4, 128, True
 autoround = AutoRoundMLLM(model, tokenizer, processor,
                           bits=bits, group_size=group_size, sym=sym)
-autoround.quantize()
-# save the quantized model, set format='auto_gptq' or 'auto_awq' to use other formats
-autoround.save_quantized(quant_path, format='auto_round', inplace=True)
+format='auto_round'
+autoround.quantize_and_save(quant_path, format=format) # quantize and save
 
 ```
 
+- known issues
+
+Several limitations currently affect offline quantized model loading in sglang:
+
+1. Mixed-bit Quantization Limitations
+
+    Mixed-bit quantization is not fully supported. Due to vLLM's layer fusion (e.g., QKV fusion), applying different bit-widths to components within the same fused layer can lead to compatibility issues.
+
+
+2. Limited Support for Quantized MoE Models
+    
```

Comment:
Most quantized MoE models may encounter inference issues due to kernel-related limitation. These issues might be resolved in future updates of sglang. If you experience any problems, consider using Hugging Face Transformers as an alternative. Detailed failure cases are listed below.